test/routes.spec.js

```diff
@@ -122,6 +123,33 @@ describe('API Routes', () => {
     });
   });
 
+  describe('GET /api/v1/waves?YEAR=:year', () => {
+    it('should return waves from a specific year', done => {
+      const mockData = {
+        'WAVE_ID': '28689',
+        'SOURCE_ID': '5586',
+        'YEAR': '2013',
+        'MONTH': '8',
+        'LOCATION': 'QUEEN\'S WHARF',
+        'MAXIMUM_HEIGHT': '0.07',
+        'FATALITIES': '',
+        'FATALITY_ESTIMATE': '',
+        'ALL_DAMAGE_MILLIONS': null,
+        'DAMAGE_ESTIMATE': '',
+      }
+
+      chai.request(server).get('/api/v1/waves?YEAR=2013')
+      .end((error, response) => {
+        const index = response.body.findIndex(obj => obj.WAVE_ID === mockData.WAVE_ID)
+
+        response.body.length.should.equal(2);
+        response.body.should.be.a('array');
+        response.body[index].should.include(mockData);
+        done()
```

Comment:
This was pretty much the exact same as the other GET request that you did, but I am getting just waves from 2013